addons/website/views/snippets/s_image_gallery.xml

```diff
@@ -57,8 +57,12 @@
                 <we-button data-select-class="o_arrows-rounded">Rounded</we-button>
             </we-select>
             <we-checkbox string="Image Cover" data-select-class="s_image_gallery_cover" data-dependencies="slideshow_mode_opt"/>
-            <we-checkbox string="Show Indicators" data-select-class="s_image_gallery_show_indicators" data-dependencies="slideshow_mode_opt"/>
-            <we-checkbox string="Rounded Indicators" data-select-class="o_indicators-rounded" data-dependencies="slideshow_mode_opt"/>
+            <we-select string="Indicators" data-dependencies="slideshow_mode_opt">
+                <we-button data-select-class="">None</we-button>
+                <we-button data-select-class="s_image_gallery_show_indicators o_indicators-dots">Dots</we-button>
+                <we-button data-select-class="s_image_gallery_show_indicators">Squared Miniatures</we-button>
+                <we-button data-select-class="s_image_gallery_show_indicators o_indicators-rounded">Rounded Miniatures</we-button>
```

Comment:
Rename your classes in your commits. Not o_incicators-rounded. Only underscore and if the option is specific to this snippet, start with the snippet name. So: s_image_gallery_indicators_rounded